starters/next-wordpress-starter/.env.example

```diff
@@ -1,6 +1,8 @@
 WPGRAPHQL_URL=
 IMAGE_DOMAIN=
 #PANTHEON_UPLOAD_PATH=/example
-#PREVIEW_SECRET=
+PREVIEW_SECRET=
```

Comment:
I like having the optional vars commented out. Maybe we can even add a note at the top explaining that?
Or we should uncomment all of them.
My vote is to comment out this and `IS_LIVE_ENVIRONMENT`